index.d.ts

```diff
@@ -335,7 +335,8 @@ declare module '@primer/components' {
   export interface SelectMenuLoadingAnimationProps extends CommonProps, Omit<React.HTMLAttributes<HTMLDivElement>, 'color'> {}
 
   interface SelectMenuItemCommonProps extends CommonProps {
-    selected?: boolean;
+    selected?: boolean
+    ref?: React.RefObject<HTMLAnchorElement> | React.RefObject<HTMLDivElement> | null
```

Comment:
Most of the time we expect people to use the default `a` tag on SelectMenu.Item, but in ✨ another project at GitHub ✨ folks are using the SelectMenu.Item as a `button`